### DISCUSSION TITLE
Who can I validate my PR ?

### DISCUSSION THREAD

**Discussion by martinez-hugo** [cdnjs/cdnjs#14181] [General]
Hi ! Who can validate my PR ? it's for a project that use CDN JS 

https://github.com/cdnjs/packages/pull/1472

Thanks for all

**Comment by MattIPv4** [comment]
Please have some patience, you opened that PR literally two hours ago...

I maintain cdnjs, unpaid, in my spare time, when I have time :)

I will look at it when I have some time, probably at the weekend.
